src/bin/scribble.ts

```diff
@@ -540,511 +581,531 @@ function pickVersion(versionUsedMap: Map<string, string>): string {
-                targetResult.compilerVersion !== undefined
-                    ? targetResult.compilerVersion
-                    : compilerVersion;
+                const compilerVersionUsed: string =
+                    targetResult.compilerVersion !== undefined
+                        ? targetResult.compilerVersion
+                        : compilerVersion;
 
-            if (compilerVersionUsed === "auto") {
-                error(
-                    `When passing in JSON you must specify an explicit compiler version with --compiler-version`
-                );
-            }
+                if (compilerVersionUsed === "auto") {
+                    error(
+                        `When passing in JSON you must specify an explicit compiler version with --compiler-version`
+                    );
+                }
 
-            const ctx = new ASTContext();
-            const reader = new ASTReader(ctx);
+                const ctx = new ASTContext();
+                const reader = new ASTReader(ctx);
 
-            if (targetResult.files.size === 0) {
-                error(
-                    `Missing source files in input. Did you pass in JSON without a sources entry?`
-                );
-            }
+                if (targetResult.files.size === 0) {
+                    error(
+                        `Missing source files in input. Did you pass in JSON without a sources entry?`
+                    );
+                }
 
-            const originalUnits = reader.read(targetResult.data, undefined, targetResult.files);
-            /**
-             * This is inefficient, but we re-create the utils source unit for every target. This is due to
-             * the inability to merge the id-spaces of the nodes of different compilation results.
-             */
-            compilerVersionUsedMap.set(target, compilerVersionUsed);
-            groupsMap.set(target, originalUnits);
-            ctxtsMap.set(target, ctx);
-            filesMap.set(target, targetResult.files);
-        } catch (e) {
-            console.error(e);
+                const originalUnits = reader.read(targetResult.data, undefined, targetResult.files);
+                /**
+                 * This is inefficient, but we re-create the utils source unit for every target. This is due to
+                 * the inability to merge the id-spaces of the nodes of different compilation results.
+                 */
+                compilerVersionUsedMap.set(target, compilerVersionUsed);
+                groupsMap.set(target, originalUnits);
+                ctxtsMap.set(target, ctx);
+                filesMap.set(target, targetResult.files);
+            } catch (e) {
+                console.error(e);
 
-            process.exit(1);
+                process.exit(1);
+            }
         }
-    }
 
-    /**
-     * Without --disarm we need to instrument and output something.
-     */
-    const contentsMap: SourceMap = new Map();
+        /**
```

Comment:
Nit: This comment is stale. Should delete

Reply:
Fixed in [839523e](https://github.com/ConsenSys/scribble/pull/143/commits/839523e6d9a55d3032a411ed5152ef90b9d21a35)